pyspecProcScripts/integral_w_error.py

```diff
@@ -27,7 +27,6 @@ def integral_w_errors(s,sig_path,error_path, indirect='vd', direct='t2',fl=None,
     """
     assert s.get_ft_prop(direct), "need to be in frequency domain!"
     frq_slice = integrate_limits(select_pathway(s,sig_path),
-            convolve_method='Gaussian',
```

Comment:
I have this function use the default for `integrate_limits` --> might make sense to add this as a kwarg for this function, which I didn't do.

Reply:
made into kwarg